File: app/src/main/java/com/appsbygu/kadai_weather_java/models/apis/Weathers/PinpointLocation.java

package com.appsbygu.kadai_weather_java.models.apis.Weathers;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

import io.realm.RealmObject;

public class PinpointLocation extends RealmObject{

    @SerializedName("link")
    @Expose
    private String link;
    @SerializedName("name")
    @Expose
    private String name;

    public String getLink() {
        return link;
    }

    public String getName() {
        return name;
    }

}

File: app/src/main/java/com/appsbygu/kadai_weather_java/models/apis/Weathers/PinpointLocation.kt
package com.appsbygu.kadai_weather_java.models.apis.Weathers

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

import io.realm.RealmObject

open class PinpointLocation : RealmObject() {

    @SerializedName("link")
    @Expose
    var link: String? = null
    @SerializedName("name")
    @Expose
    var name: String? = null

}
